examples/ipfs-add.js

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env node
 
-var ipfs = require('./index')('localhost', 5001)
```

Comment:
this file (ipfs-add) is more than an example. it's runnable. (could be a bin to install :) )